### ISSUE TITLE
Datepicker in pop-up closes pop-up without warning (when creating a m2o object from editable o2m field)

### ISSUE THREAD

**Issue by MeeleCode** [odoo/odoo#49358]
Impacted versions: 
12.0 tested (but i think it is the same with 13.0)

Steps to reproduce: 
1. Use latest Odoo v12 (I've tried used runbot) and go to project module
2. Go to tasks and open a new task
3. In new task select a project so you can see the timesheet lines
4. Create a new timesheet line and try to create a new user from m2o (using "Create and Edit...")
5. Go to Private information tab and click on Date of birth so you can see a datepicker pop-up
6. When you click on the header with day names the whole employee pop-up closes without warning

Current behavior:
This is a bug with m2o fields when they are created (and also opened) from editable o2m field. The whole pop-up will crash when you click on header of a date or datetime field in pop-up form.

Expected behavior:
The form shouldn't crash

Video/Screenshot link (optional):
[Tasks - Odoo - Popup Crash.zip](https://github.com/odoo/odoo/files/4458868/Tasks.-.Odoo.-.Popup.Crash.zip)

**Comment by pedrobaeza** [comment]
Uhm, curious and I wouldn't think about doing that sequence of steps, but I confirm it's reproducible.

@ged-odoo I think this is something for your team.

**Comment by MeeleCode** [comment]
Is there any progress regarding this bug? I know it's bizarre but it is real problem.

**Comment by Yenthe666** [comment]
@MeeleCode no, as long as there is no response or link to a PR it is probably not fixed/handled.
